solr/tests/common.py

```diff
@@ -42,23 +29,10 @@
     "solr.search_handler.request_times.mean",
     "solr.search_handler.request_times.mean_rate",
     "solr.search_handler.request_times.one_minute_rate",
+    "solr.search_handler.requests",
+    "solr.search_handler.time",
+    "solr.search_handler.timeouts",
     "solr.searcher.maxdocs",
+    "solr.searcher.numdocs",
+    "solr.searcher.warmup",
 ]
-
-SOLR_6_METRICS = [
```

Comment:
Small nit: Since these metrics are Solr 6-only, could we update their descriptions in `metadata.csv` to mention this?

Reply:
Good call we definitely need to update the CSV file but I wonder, maybe we should simply remove them now?

Reply:
I would vote for removing them